src/app/inventory/store/masterwork.ts

```diff
@@ -110,12 +110,12 @@ function buildForsakenMasterworkStats(
       typeName: null,
       typeIcon: masterworkSocket.plug.plugItem.displayProperties.icon,
       typeDesc: masterworkSocket.plug.plugItem.displayProperties.description,
-      statHash: masterwork.statTypeHash,
-      statName: statDef.displayProperties.name,
-      statValue: masterworkSocket.plug.stats
-        ? masterworkSocket.plug.stats[masterwork.statTypeHash]
-        : 0,
       tier: masterwork.value,
+      stats: {
+        hash: masterwork.statTypeHash,
+        name: statDef.displayProperties.name,
+        value: masterworkSocket.plug.stats?.[masterwork.statTypeHash] || 0,
+      }[0],
```

Comment:
i don'y think this does what you think it does unless i am really, really confused on syntax

Reply:
it puts that into index 0? without it it causes a type error.

Reply:
you're creating an object then immediately selecting index 0 from it (which it doesn't have)
![image](https://user-images.githubusercontent.com/31990469/86549607-f3cfc900-bef4-11ea-8317-c26782d99c7f.png)

i think what's intended here is 
```
      stats: [{
        hash: masterwork.statTypeHash,
        name: statDef.displayProperties.name,
        value: masterworkSocket.plug.stats?.[masterwork.statTypeHash] || 0,
      }],
```